### DISCUSSION TITLE
Is there a mapping of the winget-cli release to the DesktopAppInstaller msix version?

### DISCUSSION THREAD

**Discussion by byjrack** [microsoft/winget-cli#3952] [Q&A] [answered]
The patch looks to be in-sync so like cli 1.6.3133 maps to msix 1.21.3133.0.

Is it as easy as +5 to the minor?  Just helps me document a bit since the Get-Appx will show the msix version while the --version endpoint shows cli version and folks are getting a bit confused.

**Comment by Trenly** [comment, marked as answer]
There is not an exact mapping. @denelon discusses the reason the versions are different in the recent [Open At Microsoft - WinGet Community Repository Frequently Asked Questions](https://learn.microsoft.com/en-us/shows/open-at-microsoft/winget-community-repository-frequently-asked-questions-faqs#time=00m47s) video. The short answer is found in the [Community FAQ](https://github.com/microsoft/winget-pkgs/blob/master/FAQ.md#why-do-winget-and-appinstaller-have-different-versions) - WinGet is bundled with AppInstaller, but is a separate application from AppInstaller

**Reply by byjrack** [reply]
Cool I was trying to be a bit deterministic based on seeing the patch seemingly in lock step, but I didn't go and check them all to see if that held true.  I get the reasons that they have independent versions it was more if the version of cli influenced the version of  dai in a consistent way.

**Reply by denelon** [reply]
Yes, the same "internal" build used for App Installer is used for publishing the signed version we release here. The "patch" versions will be the same.
